src/Header.js

```diff
@@ -80,12 +81,18 @@ const Header = React.createClass({
     }
     let headerRows = [];
     this.props.headerRows.forEach((row, index) => {
+      // To allow header filters to be visible
+      let rowHeight = 'auto';
+      if (row.rowType === 'filter') {
+        rowHeight = '500px';
```

Comment:
@malonecj: This seems to be a safe height for the React-Select component (which have max-height 200px).